### PR TITLE
add a request to check & equip for new badges

### PR DESCRIPTION
**Pull request by ramarro123**
**Changes made:**

check and equip new badges
it can be splitted in 2 different requests check --> equip
but for me it doesn't make sense, since equip need the awarded badges in any case.

### CODE REVIEW

**Comment by aschlosser** [comment]
Fixes issue #188

File: src/main/java/com/pokegoapi/api/player/PlayerProfile.java [hunk 1]
```diff
@@ -188,6 +192,41 @@ public void addCurrency(String name, int amount) throws InvalidCurrencyException
 	}
 
 	/**
+	 * Check and equip badges.
+	 *
+	 * @throws InvalidCurrencyException the invalid currency exception
```
Comment (aschlosser) [inline src/main/java/com/pokegoapi/api/player/PlayerProfile.java:198]: Maybe add `@throws` for `LoginFailedException`

**Comment by ramarro123** [comment]
i accidentally delete the branch :) 
i am a noob on git, sorry.. old cvs school...

i'll wait till it get merged and apply a patch after that, or maybe during merge the merge_team will be so nice  to add it for me :)

sorry

**Comment by vmarchaud** [comment]
@ramarro123 remake a new PR with your patch, more cleaner than 2 PRs

**Comment by ramarro123** [comment]
with the power of google, i was able to switch to the branch, wasn't deleted (not in github at least) 
i think that it's good to go

**Pull request merged**